src/vast_tracker.js

```diff
@@ -319,19 +319,30 @@ export class VASTTracker extends EventEmitter {
     }
   }
 
+  /**
+   * Send a request to the URI provided by the VAST <Error> element.
+   * @param {Object} [macros={}] - An optional Object containing macros and their values to be used and replaced in the tracking calls.
+   *
+   * @param {Boolean} [isCustomCode=false] - Flag to allow custom values on error code.
+   */
+
+  error(macros = {}, isCustomCode = false) {
+    this.trackURLs(this.ad.errorURLTemplates, macros, { isCustomCode });
+  }
+
   /**
    * Send a request to the URI provided by the VAST <Error> element.
    * If an [ERRORCODE] macro is included, it will be substitute with errorCode.
+   * This method is deprecated
```

Comment:
You can use the JSDoc `deprecated` flag:  https://jsdoc.app/tags-deprecated.html